index.html

```diff
@@ -0,0 +1,112 @@
+      <span class="line"></span>
+      <span class="line"></span>
+    </div>
+  </nav>
+
+  <!-- main -->
+
+  <main class="program-description">
+    <p class="previous-title">"Get your Crypto Game ON"</p>
+    <h1 class="main-title">
+      CRYTO CONGRESS <br> GLOBAL SUMMIT <br> 2022
+    </h1>
+    <p class="description1">Crypto Congress is back for its third edition on 15 October 2022 and once again will be hosted online with controversial speakers and discussions. The event will be a festive gathering of 2,500+ attendees, uniting entrepreneurs, traders, investors, developers, industry enthusiasts, among others, with a jam-packed day with exciting keynotes, workshops, presentations, contentious topics, product exhibitions, competitions and entertainment.</p>
+    <p class="dates">2022.10.15(THU) ~ 16(FRI)</p>
+    <p class="direction">@ Radisson Hotel, Nevada, Unites States</p>
+
+  </main>
+
+  <!-- program section -->
+
+  <section class="program-section">
+    <h2 class="title2">
+      Main Cryptos
+    </h2>
+    <div class="card-container">
+      <div class="card-program">
+        <img src="./images/currencies/bitcoin.png" alt="Bitcoin cryptocurrency" class="icon-coin">
+        <h3 class="coin">Bitcoin</h3>
+        <p class="des-coin">Bitcoin is regarded as the first decentralized cryptocurrency using blockchain technology.</p>
+      </div>
+      <div class="card-program">
+        <img src="./images/currencies/etherium.png" alt="Etherium cryptocurrency" class="icon-coin">
+        <h3 class="coin">Etherium</h3>
+        <p class="des-coin"> Ethereum is a platform that uses blockchain tech to enable the creation of smart contracts.</p>
+      </div>
+      <div class="card-program">
+        <img src="./images/currencies/litecoin.png" alt="Litecoin cryptocurrency" class="icon-coin">
+        <h3 class="coin">Litecoin</h3>
+        <p class="des-coin">Litecoin is a peer-to-peer cryptocurrency released under the MIT/X11 license.</p>
+      </div>
+      <div class="card-program">
+        <img src="./images/currencies/cardano.png" alt="Cardano cryptocurrency" class="icon-coin">
+        <h3 class="coin">Cardano</h3>
+        <p class="des-coin">Cardano is a public blockchain platform. It is open-source and decentralized</p>
+      </div>
+      <div class="card-program">
+        <img src="./images/currencies/dogecoin.png" alt="Dogecoin cryptocurrency" class="icon-coin">
+        <h3 class="coin">Dogecoin</h3>
+        <p class="des-coin">Originally made as a joke poking fun of rampant speculation on cryptocurrencies.</p>
+      </div>
+    </div>
+    <div class="button-cta">
+      <a id="cta" href="#" class="join">Join cryptocurrency summit 2022</a>
+    </div>
+
+  </section>
+
+  <!-- speaker section -->
+  <section class="speakers">
+    <h2 class="featured-speakers">Featured Speaker</h2>
+    <div class="card-speaker-container">
+     
+    </div>
+    <button class="show-more">MORE</button>
+
```

Comment:
- [ ] [OPTIONAL] There is one little thing I realized is in this button. Please also change the text `MORE` to `LESS` when it opened 🙌

#### Screenshot 
![image](https://user-images.githubusercontent.com/69011963/152239498-46c37803-bd56-44ae-a73f-d9aecbad9ca8.png)